gax-java/gax/src/main/java/com/google/api/gax/rpc/Callables.java

```diff
@@ -69,12 +79,21 @@ public static <RequestT, ResponseT> UnaryCallable<RequestT, ResponseT> retrying(
 
     RetryAlgorithm<ResponseT> retryAlgorithm =
         new RetryAlgorithm<>(
-            new ApiResultRetryAlgorithm<ResponseT>(),
+            new ApiResultRetryAlgorithm<>(),
             new ExponentialRetryAlgorithm(settings.getRetrySettings(), clientContext.getClock()));
-    ScheduledRetryingExecutor<ResponseT> retryingExecutor =
+
+    return retrying(innerCallable, retryAlgorithm, clientContext);
+  }
+
+  public static <RequestT, ResponseT> ServerStreamingCallable<RequestT, ResponseT> retrying(
+      ServerStreamingCallable<RequestT, ResponseT> innerCallable,
+      ServerStreamingCallSettings<RequestT, ResponseT> callSettings,
+      StreamingRetryAlgorithm<Void> retryAlgorithm,
```

Comment:
I know `StreamingRetryAlgorithm<Void>` always have a `Void` type before, do you know if it's by design? Asking because I think this method is only being called directly from Bigtable, it would be better to change it to `ResponseT`, but it may have other implications.

Reply:
I think it's because RetryingServerStreamingCallable uses a ScheduledRetryingExecutor<Void>. It doesn't expect a ApiFuture<responseT> in the executor.createFuture() like the unary calls.